backend-lib/src/main/rust/lib.rs

```diff
@@ -828,7 +829,7 @@ pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_listTrans
         let db_data = wallet_db(env, network, db_data)?;
         let account = account_id_from_jni(env, account_uuid)?;
 
-        match db_data.get_transparent_receivers(account, true) {
+        match db_data.get_transparent_receivers(account, true, true) {
```

Comment:
This should match the Swift SDK:
```suggestion
        // Zashi does not support standalone keys, so we do not request standalone receivers.
        match db_data.get_transparent_receivers(account, true, false) {
```